cmd/cli/serve/serve.go

```diff
@@ -61,109 +53,19 @@ var (
-		LimitJobMemory:         "",
-		LimitJobGPU:            "",
-		MaxJobExecutionTimeout: model.NoJobTimeout,
-		PrivateInternalIPFS:    true,
-	}
-}
-
-func SetupCapacityManagerCLIFlags(cmd *cobra.Command, OS *ServeOptions) {
-	cmd.PersistentFlags().StringVar(
-		&OS.LimitTotalCPU, "limit-total-cpu", OS.LimitTotalCPU,
-		`Total CPU core limit to run all jobs (e.g. 500m, 2, 8).`,
-	)
-	cmd.PersistentFlags().StringVar(
-		&OS.LimitTotalMemory, "limit-total-memory", OS.LimitTotalMemory,
-		`Total Memory limit to run all jobs  (e.g. 500Mb, 2Gb, 8Gb).`,
-	)
-	cmd.PersistentFlags().StringVar(
-		&OS.LimitTotalGPU, "limit-total-gpu", OS.LimitTotalGPU,
-		`Total GPU limit to run all jobs (e.g. 1, 2, or 8).`,
-	)
-	cmd.PersistentFlags().StringVar(
-		&OS.LimitJobCPU, "limit-job-cpu", OS.LimitJobCPU,
-		`Job CPU core limit for single job (e.g. 500m, 2, 8).`,
-	)
-	cmd.PersistentFlags().StringVar(
-		&OS.LimitJobMemory, "limit-job-memory", OS.LimitJobMemory,
-		`Job Memory limit for single job  (e.g. 500Mb, 2Gb, 8Gb).`,
-	)
-	cmd.PersistentFlags().StringVar(
-		&OS.LimitJobGPU, "limit-job-gpu", OS.LimitJobGPU,
-		`Job GPU limit for single job (e.g. 1, 2, or 8).`,
-	)
-	cmd.PersistentFlags().DurationVar(
-		&OS.MaxJobExecutionTimeout, "max-timeout", OS.MaxJobExecutionTimeout,
-		`Job time exeuciton limit for a single job (e.g. 30m, 1hr)`,
-	)
-	cmd.PersistentFlags().StringSliceVar(
-		&OS.JobExecutionTimeoutClientIDBypassList, "job-execution-timeout-bypass-client-id", OS.JobExecutionTimeoutClientIDBypassList,
-		`List of IDs of clients that are allowed to bypass the job execution timeout check`,
-	)
-}
-
-func SetupLibp2pCLIFlags(cmd *cobra.Command, OS *ServeOptions) {
-	cmd.PersistentFlags().StringVar(
-		&OS.PeerConnect, "peer", OS.PeerConnect,
-		`A comma-separated list of libp2p multiaddress to connect to. `+
-			`Use "none" to avoid connecting to any peer, `+
-			`"env" to connect to the default peer list of your active environment (see BACALHAU_ENVIRONMENT env var).`,
-	)
-	cmd.PersistentFlags().StringVar(
-		&OS.HostAddress, "host", OS.HostAddress,
-		`The host to listen on (for both api and swarm connections).`,
-	)
-	cmd.PersistentFlags().IntVar(
-		&OS.SwarmPort, "swarm-port", OS.SwarmPort,
-		`The port to listen on for swarm connections.`,
+func GetPeers(peerConnect string) ([]multiaddr.Multiaddr, error) {
+	var (
+		peersStrings []string
 	)
-}
-
-func GetPeers(OS *ServeOptions) ([]multiaddr.Multiaddr, error) {
-	var peersStrings []string
-	if OS.PeerConnect == DefaultPeerConnect {
-		peersStrings = []string{}
-	} else if OS.PeerConnect == "env" {
-		peersStrings = system.Envs[system.GetEnvironment()].BootstrapAddresses
+	// TODO(forrest): [ux] this is a really confusing way to configure bootstrap peers.
+	// The convenience is nice by passing a single 'env' value, and can be improved with sane defaults commented
+	// out in the config. If a user wants to connect then can pass the --peer flag or uncomment the config values.
```

Comment:
Any opinions on this? I'd like to leave this unchanged in this PR, and make a change if we are up for it, in a follow on.

Reply:
I think we should definitely leave bootstrap config until later